bootstrap/handlers/messaging_test.go

```diff
@@ -39,18 +39,6 @@ func TestMain(m *testing.M) {
 }
 
 func TestBootstrapHandler(t *testing.T) {
-	validCreateClientSecure := config.MessageBusInfo{
-		Type:       messaging.Redis,
-		Protocol:   "redis",
-		Host:       "localhost",
-		Port:       6379,
-		AuthMode:   boostrapMessaging.AuthModeUsernamePassword,
-		SecretName: "redisdb",
-	}
```

Comment:
please modify the message bus config instead of removing the unit test
```
    Protocol: "mqtt",
    Host: "localhost",
    Port: 1883,
    Type: messaging.MQTT,
    AuthMode: boostrapMessaging.AuthModeUsernamePassword,
    SecretName: "mqtt-bus"
```

Reply:
Per discussion, simply remove these unit tests as these removed unit tests are verifying `MessagingBootstrapHandler` when message bus type is Redis Pub/Sub.
These unit tests will fail when message bus type is mqtt. This is because `MessagingBootstrapHandler` will [try to invoke Connect()](https://github.com/edgexfoundry/go-mod-bootstrap/blob/v4.0.0-dev.3/bootstrap/handlers/messaging.go#L81) and Redis message client actually simply return nil in its Connect() while MQTT message client needs to access to a valid MQTT broker.